benches/fs_read_write.rs

```diff
@@ -33,68 +35,101 @@ pub fn run_benchmark(c: &mut Criterion) {
 
 
 
-        // --- COMPARING read + write w/o lssek CALLS ACROSS Lind + Native OS kernel ---
+    // --- COMPARING read + write w/o lseek CALLS ACROSS Lind + Native OS kernel ---
     // This is separated because writeat and readat do a lot of seeking.  It's
     // useful to have a comparison which does not.
     let mut group = c.benchmark_group("Compare fs:write+read");
 
     // Should be similar.  Use a linear scale...
     group.plot_config(criterion::PlotConfiguration::default().summary_scale(criterion::AxisScale::Linear));
 
-    let fd = cage.open_syscall("foo",O_CREAT | O_TRUNC | O_WRONLY,S_IRWXA);
-    // Let's see how fast various file system calls are
-    group.bench_function("TF02: Lind write", |b| b.iter(||
-        {
-            let _ = cage.write_syscall(fd,tests::str2cbuf("Well, hello there!!!"),20);
-        }
-    ));
 
-    cage.lseek_syscall(fd,0,SEEK_SET);
 
-    group.bench_function("TF02: Lind read", |b| b.iter(||
-        {
-            let mut read_buffer = tests::sizecbuf(20);
-            cage.read_syscall(fd,read_buffer.as_mut_ptr(), 20);
-        }
-    ));
+    // First do this for Lind
 
-    cage.close_syscall(fd);
-    cage.unlink_syscall("foo");
+    // Reduce the time to reduce disk space needed and go faster.  
+    // Default is 5s...
+    group.measurement_time(Duration::from_secs(2));
```

Comment:
If we're going to configure these groups the same for every benchmark it might make sense to factor something out for reuse.

Reply:
We're not.  It is just for these functions.  (And there is already a [place where the refactored bits go](https://github.com/Lind-Project/safeposix-rust/blob/develop/benches/global_criterion_settings.rs).)